src/main/java/com/minecolonies/core/colony/buildings/AbstractBuildingContainer.java

```diff
@@ -159,7 +159,7 @@ public void registerBlockPosition(@NotNull final Block block, @NotNull final Blo
             if (entity instanceof TileEntityColonyBuilding buildingEntity)
             {
                 buildingEntity.setStructurePack(StructurePacks.getStructurePack(getStructurePack()));
-                buildingEntity.setRotationMirror(getRotationMirror());
```

Comment:
you sure this works for now? :D

Reply:
Until we do the full convoluted 3d chess checking 2 old blueprints kind of thing yes.